bolt/test/X86/shared_object_entry.s

```diff
@@ -2,7 +2,7 @@
 # RUN: ld.lld %t.o -o %t.so --shared --entry=func1.cold.1 --emit-relocs
 # RUN: llvm-bolt -relocs %t.so -o %t -reorder-functions=hfsort+ \
 # RUN:    -split-functions -reorder-blocks=ext-tsp -split-all-cold \
-# RUN:    -dyno-stats -icf=1 -use-gnu-stack
```

Comment:
To avoid breaking existing users, I’d suggest adding `1` as alias of unsafe/`all`, with a deprecation warning.

Reply:
I don't think llvm guarantees backward compatibility with options.

Reply:
Then please update all the places where we have `icf=1` ourselves (internal uses, in-tree tests, binary tests). But I think it's better not to screw our users without a warning.